src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  width: 200px;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+    margin: 40px 16px 0;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-color;
+    margin: 4px 0 0 16px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: row;
+    box-sizing: border-box;
+    justify-content: space-between;
+
+    margin: 16px 16px 0;
```

Comment:
The `&__reviews` block should have a margin that follows the design specifications. Ensure that the margin is consistent with the design layout.